packages/pg-cluster/src/Cluster.ts

```diff
@@ -0,0 +1,109 @@
+import {Readable} from 'stream';
+import {
+  Connection,
+  pgFormat,
+  Queryable,
+  QueryableType,
+  Transaction,
+} from '@databases/pg';
+import AbortSignal from '@databases/pg/lib/types/AbortSignal';
+import TransactionOptions from '@databases/pg/lib/types/TransactionOptions';
+import sql, {SQLQuery} from '@databases/sql';
```

Comment:
To reduce the risk of multiple versions of `@databases/sql` being added to the same project, you can import the versions of `sql` and `SQLQuery` that are exported by `@databases/pg`:

```suggestion
  sql,
  SQLQuery,
} from '@databases/pg';
import AbortSignal from '@databases/pg/lib/types/AbortSignal';
import TransactionOptions from '@databases/pg/lib/types/TransactionOptions';
```